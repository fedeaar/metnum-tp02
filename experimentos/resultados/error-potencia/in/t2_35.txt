5.791199000000000000e+06 -3.314104000000000000e+06 6.939580000000000000e+05 9.396230000000000000e+05 -1.201065000000000000e+06 1.319500000000000000e+04 -3.269337000000000000e+06 -4.483950000000000000e+05 9.647610000000000000e+05 -3.135510000000000000e+06 -2.118784000000000000e+06 2.249780000000000000e+05 1.088150000000000000e+05 -1.807457000000000000e+06 -4.925570000000000000e+05 -1.992778000000000000e+06 9.973070000000000000e+05 -9.203390000000000000e+05 1.543000000000000000e+05 1.502880000000000000e+05
-3.314104000000000000e+06 4.431143000000000000e+06 6.575550000000000000e+05 1.860530000000000000e+05 -4.172400000000000000e+05 1.675160000000000000e+05 2.418890000000000000e+06 -2.427180000000000000e+05 -2.037843000000000000e+06 1.771895000000000000e+06 1.970940000000000000e+06 -1.362676000000000000e+06 1.897330000000000000e+05 3.492874000000000000e+06 4.026120000000000000e+05 1.902735000000000000e+06 -1.617500000000000000e+05 6.317300000000000000e+04 7.072490000000000000e+05 2.531560000000000000e+05
6.939580000000000000e+05 6.575550000000000000e+05 7.622899000000000000e+06 7.216020000000000000e+05 -7.009200000000000000e+04 -1.164629000000000000e+06 1.351900000000000000e+05 8.472790000000000000e+05 -1.196586000000000000e+06 -1.430110000000000000e+05 1.047970000000000000e+05 6.722070000000000000e+05 2.016105000000000000e+06 6.118780000000000000e+05 1.348580000000000000e+05 1.869012000000000000e+06 2.759561000000000000e+06 -4.175410000000000000e+05 7.118810000000000000e+05 1.003790000000000000e+06
9.396230000000000000e+05 1.860530000000000000e+05 7.216020000000000000e+05 6.745058000000000000e+06 -6.963340000000000000e+05 -1.159520000000000000e+05 -2.705297000000000000e+06 1.139900000000000000e+05 3.413433000000000000e+06 -1.126017000000000000e+06 -4.940050000000000000e+05 -1.263690000000000000e+06 2.717574000000000000e+06 -8.095340000000000000e+05 8.737010000000000000e+05 8.704270000000000000e+05 2.639730000000000000e+06 -2.373550000000000000e+05 2.465187000000000000e+06 1.022370000000000000e+05
-1.201065000000000000e+06 -4.172400000000000000e+05 -7.009200000000000000e+04 -6.963340000000000000e+05 5.164772000000000000e+06 5.675190000000000000e+05 1.996430000000000000e+06 -1.064751000000000000e+06 -1.893951000000000000e+06 4.293810000000000000e+05 1.038917000000000000e+06 1.092697000000000000e+06 8.505170000000000000e+05 -8.793600000000000000e+05 -1.428330000000000000e+05 -1.133216000000000000e+06 -1.059480000000000000e+05 2.509830000000000000e+05 7.354420000000000000e+05 4.580150000000000000e+05
1.319500000000000000e+04 1.675160000000000000e+05 -1.164629000000000000e+06 -1.159520000000000000e+05 5.675190000000000000e+05 4.615123000000000000e+06 3.404208000000000000e+06 -1.302450000000000000e+05 2.643080000000000000e+05 8.647110000000000000e+05 -7.520960000000000000e+05 -1.529759000000000000e+06 -1.728075000000000000e+06 -9.493790000000000000e+05 -2.431990000000000000e+05 -2.669497000000000000e+06 -5.621570000000000000e+05 -2.834800000000000000e+04 -2.282459000000000000e+06 1.103459000000000000e+06
-3.269337000000000000e+06 2.418890000000000000e+06 1.351900000000000000e+05 -2.705297000000000000e+06 1.996430000000000000e+06 3.404208000000000000e+06 8.772785000000000000e+06 -2.354286000000000000e+06 -1.741169000000000000e+06 1.549927000000000000e+06 8.333690000000000000e+05 -2.206663000000000000e+06 3.678560000000000000e+05 6.068310000000000000e+05 -4.793700000000000000e+05 -1.623751000000000000e+06 -2.759028000000000000e+06 -2.291680000000000000e+05 -2.124957000000000000e+06 1.633130000000000000e+06
-4.483950000000000000e+05 -2.427180000000000000e+05 8.472790000000000000e+05 1.139900000000000000e+05 -1.064751000000000000e+06 -1.302450000000000000e+05 -2.354286000000000000e+06 8.309562000000000000e+06 -9.259670000000000000e+05 1.576497000000000000e+06 3.946930000000000000e+05 3.313120000000000000e+05 -1.919540000000000000e+05 2.408450000000000000e+05 1.342300000000000000e+05 1.400357000000000000e+06 8.231500000000000000e+04 2.192118000000000000e+06 3.501350000000000000e+05 7.425240000000000000e+05
9.647610000000000000e+05 -2.037843000000000000e+06 -1.196586000000000000e+06 3.413433000000000000e+06 -1.893951000000000000e+06 2.643080000000000000e+05 -1.741169000000000000e+06 -9.259670000000000000e+05 6.816597000000000000e+06 1.226640000000000000e+05 -4.729840000000000000e+05 -8.980360000000000000e+05 1.280865000000000000e+06 -2.338949000000000000e+06 7.329810000000000000e+05 4.007550000000000000e+05 1.626840000000000000e+05 7.249920000000000000e+05 -1.897777000000000000e+06 -1.352714000000000000e+06
-3.135510000000000000e+06 1.771895000000000000e+06 -1.430110000000000000e+05 -1.126017000000000000e+06 4.293810000000000000e+05 8.647110000000000000e+05 1.549927000000000000e+06 1.576497000000000000e+06 1.226640000000000000e+05 6.887653000000000000e+06 2.983042000000000000e+06 -8.315500000000000000e+05 -1.529734000000000000e+06 6.772700000000000000e+04 5.597650000000000000e+05 1.574546000000000000e+06 -9.263740000000000000e+05 2.027993000000000000e+06 -2.267126000000000000e+06 -1.781873000000000000e+06
-2.118784000000000000e+06 1.970940000000000000e+06 1.047970000000000000e+05 -4.940050000000000000e+05 1.038917000000000000e+06 -7.520960000000000000e+05 8.333690000000000000e+05 3.946930000000000000e+05 -4.729840000000000000e+05 2.983042000000000000e+06 6.636635000000000000e+06 9.981600000000000000e+04 1.553211000000000000e+06 3.164650000000000000e+05 -1.348970000000000000e+05 1.372284000000000000e+06 -2.787870000000000000e+05 -3.909030000000000000e+05 -1.416520000000000000e+05 -1.293019000000000000e+06
2.249780000000000000e+05 -1.362676000000000000e+06 6.722070000000000000e+05 -1.263690000000000000e+06 1.092697000000000000e+06 -1.529759000000000000e+06 -2.206663000000000000e+06 3.313120000000000000e+05 -8.980360000000000000e+05 -8.315500000000000000e+05 9.981600000000000000e+04 8.162670000000000000e+06 1.528558000000000000e+06 -6.671220000000000000e+05 -1.798420000000000000e+05 -1.168790000000000000e+05 1.990263000000000000e+06 -3.758840000000000000e+05 2.433662000000000000e+06 9.537360000000000000e+05
1.088150000000000000e+05 1.897330000000000000e+05 2.016105000000000000e+06 2.717574000000000000e+06 8.505170000000000000e+05 -1.728075000000000000e+06 3.678560000000000000e+05 -1.919540000000000000e+05 1.280865000000000000e+06 -1.529734000000000000e+06 1.553211000000000000e+06 1.528558000000000000e+06 8.948369000000000000e+06 2.109470000000000000e+05 1.553824000000000000e+06 1.832179000000000000e+06 1.456690000000000000e+05 -1.737523000000000000e+06 2.848911000000000000e+06 8.749530000000000000e+05
-1.807457000000000000e+06 3.492874000000000000e+06 6.118780000000000000e+05 -8.095340000000000000e+05 -8.793600000000000000e+05 -9.493790000000000000e+05 6.068310000000000000e+05 2.408450000000000000e+05 -2.338949000000000000e+06 6.772700000000000000e+04 3.164650000000000000e+05 -6.671220000000000000e+05 2.109470000000000000e+05 1.040081900000000000e+07 2.935263000000000000e+06 5.884520000000000000e+05 5.339580000000000000e+05 6.475800000000000000e+04 2.218257000000000000e+06 1.323795000000000000e+06
-4.925570000000000000e+05 4.026120000000000000e+05 1.348580000000000000e+05 8.737010000000000000e+05 -1.428330000000000000e+05 -2.431990000000000000e+05 -4.793700000000000000e+05 1.342300000000000000e+05 7.329810000000000000e+05 5.597650000000000000e+05 -1.348970000000000000e+05 -1.798420000000000000e+05 1.553824000000000000e+06 2.935263000000000000e+06 5.760595000000000000e+06 1.551562000000000000e+06 1.402254000000000000e+06 -8.884930000000000000e+05 -1.363197000000000000e+06 1.705313000000000000e+06
-1.992778000000000000e+06 1.902735000000000000e+06 1.869012000000000000e+06 8.704270000000000000e+05 -1.133216000000000000e+06 -2.669497000000000000e+06 -1.623751000000000000e+06 1.400357000000000000e+06 4.007550000000000000e+05 1.574546000000000000e+06 1.372284000000000000e+06 -1.168790000000000000e+05 1.832179000000000000e+06 5.884520000000000000e+05 1.551562000000000000e+06 6.702370000000000000e+06 4.963050000000000000e+05 -1.330220000000000000e+05 -3.900060000000000000e+05 -1.103462000000000000e+06
9.973070000000000000e+05 -1.617500000000000000e+05 2.759561000000000000e+06 2.639730000000000000e+06 -1.059480000000000000e+05 -5.621570000000000000e+05 -2.759028000000000000e+06 8.231500000000000000e+04 1.626840000000000000e+05 -9.263740000000000000e+05 -2.787870000000000000e+05 1.990263000000000000e+06 1.456690000000000000e+05 5.339580000000000000e+05 1.402254000000000000e+06 4.963050000000000000e+05 6.680355000000000000e+06 -1.543075000000000000e+06 2.574434000000000000e+06 2.663325000000000000e+06
-9.203390000000000000e+05 6.317300000000000000e+04 -4.175410000000000000e+05 -2.373550000000000000e+05 2.509830000000000000e+05 -2.834800000000000000e+04 -2.291680000000000000e+05 2.192118000000000000e+06 7.249920000000000000e+05 2.027993000000000000e+06 -3.909030000000000000e+05 -3.758840000000000000e+05 -1.737523000000000000e+06 6.475800000000000000e+04 -8.884930000000000000e+05 -1.330220000000000000e+05 -1.543075000000000000e+06 4.502505000000000000e+06 -1.456708000000000000e+06 -6.108890000000000000e+05
1.543000000000000000e+05 7.072490000000000000e+05 7.118810000000000000e+05 2.465187000000000000e+06 7.354420000000000000e+05 -2.282459000000000000e+06 -2.124957000000000000e+06 3.501350000000000000e+05 -1.897777000000000000e+06 -2.267126000000000000e+06 -1.416520000000000000e+05 2.433662000000000000e+06 2.848911000000000000e+06 2.218257000000000000e+06 -1.363197000000000000e+06 -3.900060000000000000e+05 2.574434000000000000e+06 -1.456708000000000000e+06 7.353629000000000000e+06 1.237808000000000000e+06
1.502880000000000000e+05 2.531560000000000000e+05 1.003790000000000000e+06 1.022370000000000000e+05 4.580150000000000000e+05 1.103459000000000000e+06 1.633130000000000000e+06 7.425240000000000000e+05 -1.352714000000000000e+06 -1.781873000000000000e+06 -1.293019000000000000e+06 9.537360000000000000e+05 8.749530000000000000e+05 1.323795000000000000e+06 1.705313000000000000e+06 -1.103462000000000000e+06 2.663325000000000000e+06 -6.108890000000000000e+05 1.237808000000000000e+06 4.597452000000000000e+06
