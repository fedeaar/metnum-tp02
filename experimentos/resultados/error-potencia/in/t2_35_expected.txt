2.221500458672181889e+07
1.898028661246280745e+07
1.501962363547288813e+07
1.243763685294142365e+07
1.178262404720355757e+07
9.329814298840353265e+06
7.829683226008069701e+06
7.586906167950195260e+06
7.267741485540850088e+06
5.016696585027228110e+06
4.713787005647750571e+06
4.224632823849979788e+06
2.515198104476830456e+06
1.965397916819665814e+06
1.694750254672705894e+06
1.300717500035105040e+06
6.626466035604502540e+05
2.235665159896624682e+05
9.134761848607704451e+04
4.412815829260711325e+04
